test/parallel/test-cluster-master-error.js

```diff
@@ -97,30 +85,20 @@ if (cluster.isWorker) {
 
   // When cluster is dead
   master.on('exit', common.mustCall((code) => {
```

Comment:
Since you're making changes in here, may as well also rename the `master` variable to something else?